src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```

Comment:
You have margin-left here and margin-right in nav class. Maybe, you can move them to header and get there margin (or padding) 0 50px

Reply:
It's the same result. In this case, it is better for me to work with a picture and not a header.

Reply:
But I did like you said (padding in header, it's better - agree)